src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -455,21 +492,70 @@ void positiveAnonymiseEventByIds() {
-        verify(eventService, times(1)).getEventByEveId(2);
-        verify(eventService, times(1)).getEventByEveId(3);
-        verify(eventService, times(1)).getEventByEveId(4);
+        verify(eventService).getEventByEveId(1);
+        verify(eventService).getEventByEveId(2);
+        verify(eventService).getEventByEveId(3);
+        verify(eventService).getEventByEveId(4);
 
-        verify(eventService, times(1)).saveEvent(event1);
-        verify(eventService, times(1)).saveEvent(event2);
-        verify(eventService, times(1)).saveEvent(event3);
+        verify(eventService).saveEvent(event1);
+        verify(eventService).saveEvent(event2);
+        verify(eventService).saveEvent(event3);
         verifyNoMoreInteractions(eventService);
+        verify(dataAnonymisationService).registerDataAnonymisation(userAccount, event1, isManuallyRequested);
+        verify(dataAnonymisationService).registerDataAnonymisation(userAccount, event2, isManuallyRequested);
+        verify(dataAnonymisationService).registerDataAnonymisation(userAccount, event3, isManuallyRequested);
+    }
+
+    @ParameterizedTest(name = "Register data anonymisation event with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void registerDataAnonymisationEvent(boolean isManuallyRequested) {
+        UserAccountEntity userAccount = mock(UserAccountEntity.class);
+        EventEntity eventEntity = mock(EventEntity.class);
+        OffsetDateTime currentTime = OffsetDateTime.now();
+        when(currentTimeHelper.currentOffsetDateTime()).thenReturn(currentTime);
+        dataAnonymisationService.registerDataAnonymisation(userAccount, eventEntity, isManuallyRequested);
+
+        ArgumentCaptor<DataAnonymisationEntity> dataAnonymisationEntityArgumentCaptor = ArgumentCaptor.forClass(DataAnonymisationEntity.class);
+        verify(dataAnonymisationRepository).save(dataAnonymisationEntityArgumentCaptor.capture());
+        DataAnonymisationEntity dataAnonymisationEntity = dataAnonymisationEntityArgumentCaptor.getValue();
+        assertThat(dataAnonymisationEntity.getEvent()).isEqualTo(eventEntity);
+        assertThat(dataAnonymisationEntity.getTranscriptionComment()).isNull();
+        assertThat(dataAnonymisationEntity.getIsManualRequest()).isEqualTo(isManuallyRequested);
+        assertThat(dataAnonymisationEntity.getRequestedBy()).isEqualTo(userAccount);
+        assertThat(dataAnonymisationEntity.getRequestedTs()).isEqualTo(currentTime);
+        assertThat(dataAnonymisationEntity.getApprovedBy()).isEqualTo(userAccount);
+        assertThat(dataAnonymisationEntity.getApprovedTs()).isEqualTo(currentTime);
+        verify(currentTimeHelper, times(1)).currentOffsetDateTime();
+    }
+
+    @ParameterizedTest(name = "Register data anonymisation transcription comment with isManuallyRequested = {0}")
+    @ValueSource(booleans = {true, false})
+    void registerDataAnonymisationTranscriptionComment(boolean isManuallyRequested) {
```

Comment:
Can you amend the name to use the new convention please

Reply:
Updated